Code/Tools/TestImpactFramework/Runtime/Common/Code/Source/Process/Scheduler/TestImpactProcessSchedulerBus.cpp

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Process/Scheduler/TestImpactProcessSchedulerBus.h>
+
+namespace TestImpact
+{
+    ProcessCallbackResult GetAggregateProcessCallbackResult(const AZ::EBusAggregateResults<ProcessCallbackResult>& results)
+    {
+        return std::count(results.values.begin(), results.values.end(), ProcessCallbackResult::Abort)
```

Comment:
nit: It might be clearer to have an explicit `> 0` check after `count` to make this clearer

Reply:
Fixed 👍